x-pack/plugins/security_solution/cypress/integration/value_lists/value_lists.spec.ts

```diff
@@ -30,10 +30,13 @@ import {
 
 describe('value lists', () => {
   describe('management modal', () => {
+    before(() => {
+      login();
+    });
     beforeEach(() => {
-      loginAndWaitForPageWithoutDateRange(ALERTS_URL);
-      waitForListsIndexToBeCreated();
-      goToManageAlertsDetectionRules();
+      createListsIndex();
```

Comment:
Just wonder how that worked before `createListsIndex` was introduced. Looks like `waitForListsIndexToBeCreated`  was just a GET with awaiting :). 
Thank you for catching this issues!